docs/2.guide/3.going-further/7.layers.md

```diff
@@ -111,8 +111,8 @@ export default defineNuxtConfig({
 If you want to extend a private remote source, you need to add the environment variable `GIGET_AUTH=<token>` to provide a token.
 ::
 
-::callout{color="amber" icon="i-ph-warning-duotone"}
-Currently, with git remote sources, if a layer has npm dependencies, you will need to manually install them in the target project. We are working on this to auto-install layer dependencies with git sources.
+::callout{color="blue" icon="i-ph-info-duotone"}
+When using git remote sources and a layer has npm dependencies, you don't need to manually install them in the target project anymore since Nuxt v3.8.0 (See [c12 release notes](https://github.com/unjs/c12/releases/tag/v1.5.1) for full details). Before, you'd have to install the dependencies manually in the target project.
```

Comment:
```suggestion
When using git remote sources, if a layer has npm dependencies, you no longer need to manually install them in the target project since Nuxt 3.8.0 (read more in [c12 release notes](https://github.com/unjs/c12/releases/tag/v1.5.1)).
```